rfcs/0009-code-review.md

```diff
@@ -0,0 +1,173 @@
+---
+created-on: 2021-04-27
+freshness: 2021-04-27
+owner: @dj2
+status: proposal
+tags:
+  - software development
+  - practices
+---
+
+[La version française suit.](#résumé)
+
+# Code Review
+
+## Summary
+Code review is an integral part of how we work at CDS. Reviewing code
+provides an opportunity for other developers to understand the code. It
```

Comment:
Nitpicky but "code" is used a lot in this sentence / paragraph. Maybe just "to understand it" here?

Reply:
Rewrote the whole paragraph, better?

Reply:
Very nice!